doc/lightningd-config.5.txt

```diff
@@ -278,10 +278,13 @@ or precisely control where to bind and what to announce with the
     can still connect to Unix domain sockets manually).  This also disables
     all DNS lookups, to avoid leaking information.
 
+*disable-dns*::
+    Disable the DNS bootstrapping mechanism to find a node by its node ID.
+
```

Comment:
I generally prefer options to be xxx=<bool>, since you can override them (eg. suppress in the config file, enable on the cmdline). But re-enabling DNS seems weird, so it's OK.